src/backend/core/api/viewsets.py

```diff
@@ -165,6 +184,12 @@ class RoomViewSet(
     permission_classes = [permissions.RoomPermissions]
     queryset = models.Room.objects.all()
     serializer_class = serializers.RoomSerializer
+    session_manager = RecordingSessionManager(
+        worker_class=LivekitEgressWorker,
+        output_folder="recordings",
+        server_configurations=settings.LIVEKIT_CONFIGURATION,
+        enable_output_logging=settings.LOG_RECORDING_OUTPUT,
```

Comment:
I would move this inside the `LivekitEgressWorker` :

``` python
class LivekitEgressWorker(AbstractRecordingWorker):

    def __init__(self):
        self._output_folder = "recordings"  # might be a setting too?
        self._server_configurations = settings.LIVEKIT_CONFIGURATION
        self._enable_output_logging = settings.LOG_RECORDING_OUTPUT
```

because it highly relies on the "worker kind".